tachyon/device/gpu/gpu_memory.h

```diff
@@ -173,14 +173,14 @@ class GpuMemory {
   gpuStream_t stream() const { return stream_; }
   GpuMemoryType memory_type() const { return memory_type_; }
 
-  T& operator*() { return *get(); }
-  const T& operator*() const { return *get(); }
+  T& operator*() { return *data(); }
+  const T& operator*() const { return *data(); }
 
-  T* operator->() { return get(); }
-  const T* operator->() const { return get(); }
+  T* operator->() { return data(); }
+  const T* operator->() const { return data(); }
 
-  T* get() { return ptr_; }
-  const T* get() const { return ptr_; }
```

Comment:
Since you only rename `get()` to `data()` I think you can reword the commit to 
```
refac(device): rename `GpuMemory::get()` to `GpuMemory::data()`
```

Reply:
This is still not fixed btw.

Reply:
I think it's fixed but "refac(device):" is written twice. Please fix!